packages/grid/x-data-grid/src/hooks/features/rows/useGridParamsApi.ts

```diff
@@ -24,16 +25,10 @@ function warnMissingColumn(field) {
   warnedOnceMissingColumn = true;
 }
 
-let warnedOnceGetValue = false;
-function warnGetValue() {
-  console.warn(
-    [
-      `MUI: You are calling getValue. This method is deprecated and will be removed in the next major version.`,
-      `Instead, you can access the data from \`params.row}\`.`,
-    ].join('\n'),
-  );
-  warnedOnceGetValue = true;
-}
+const getCellValueWarning = buildWarning([
+  `MUI: You are calling getValue. This method is deprecated and will be removed in the next major version.`,
+  `Instead, you can access the data from \`params.row}\`.`,
```

Comment:
Not sure if it's typo or I'm missing something
```suggestion
  'Instead, you can access the data from `params.row`.',
```